src/de/mod.rs

```diff
@@ -74,6 +75,7 @@ pub enum Error {
 
     /// Error with a custom message that was preserved.
     #[cfg(feature = "custom-error-messages")]
+    #[cfg_attr(feature = "defmt", defmt(defmt::Debug2Format))]
     CustomErrorWithMessage(heapless::String<64>),
```

Comment:
As per https://defmt.ferrous-systems.com/format.html#uncompressed-adapters, I think maybe this should be:

```suggestion
    CustomErrorWithMessage(#[cfg_attr(feature = "defmt", defmt(defmt::Debug2Format))] heapless::String<64>),
```

Reply:
Ah yes, quite true!